WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -790,6 +940,22 @@ protected void onPostExecute(Boolean result) {
                 return;
             }
 
+            if (mDirectOperation != null) {
+                switch (mDirectOperation) {
+                    case COMMENT_JUMP:
+                    case COMMENT_REPLY:
+                    case COMMENT_LIKE:
+                        ReaderActivityLauncher.showReaderComments(getActivity(), mPost.blogId, mPost.postId,
```

Comment:
Why are we showing this activity and then showing comments instead of simply showing the comments directly?